server/src/data/getApps.js

```diff
@@ -50,6 +50,9 @@ const getApps = (
                 builder.where((builder) => {
                     builder.where('name', 'ilike', `%${query}%`)
                     builder.orWhere('organisation', 'ilike', `%${query}%`)
+                    if (query.match(/plugin/i)) {
```

Comment:
an obscure way to find plugins - by searching for the exact word "plugin". A backdoor until we decide how/if we expose finding plugins in the UI.